### PR TITLE
Update index.md file to add an example format for private repositories

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -166,6 +166,10 @@ For example:
 - `github.com/turbot/steampipe-plugin-aws//**/*.tf?ref=fix_7677` matches all Terraform configuration files in the specific tag of a repository.
 - `github.com/turbot/steampipe-plugin-aws//aws-test/tests/aws_acm_certificate//*.tf` matches all Terraform configuration files in the specified folder path.
 
+For private repositories the format should be as below:
+
+- `git::ssh://git@github.com/test_org/test_repo//*.tf`
```
[the remark this answers is not in the document]
Comment (Subhajit97) [reply docs/index.md:171]: @cbruno10 We had not tested this particular format, but we tested downloading the files from the private repositories during the implementation. For  example:
I have a private repository: `github.com/turbotio/ted`

Using the `paths` as below:
```
github.com/turbotio/ted///
```
I am able to download all the files in that repository.

[the remark this answers is not in the document]
Comment (cbruno10) [reply docs/index.md:171]: @bigdatasourav Were you able to get files from a private repository you have access to using the format in the current docs, like @Subhajit97 had mentioned above?

[the remark this answers is not in the document]
Comment (bigdatasourav) [reply docs/index.md:171]: @cbruno10 I am not able to access the private repo with the above format - 
-- paths = [ "github.com/bigdatasourav/test2///"]  // not working, query was running for an indefinite period, had to force close it.

-- paths = [ "git::ssh://git@github.com/bigdatasourav/test2///"] // working fine

-- paths = [ "git@github.com/bigdatasourav/test2///"] // err - failed to get directory specified by the source

File: docs/index.md [hunk 2]
```diff
@@ -166,6 +166,10 @@ For example:
 - `github.com/turbot/steampipe-plugin-aws//**/*.tf?ref=fix_7677` matches all Terraform configuration files in the specific tag of a repository.
 - `github.com/turbot/steampipe-plugin-aws//aws-test/tests/aws_acm_certificate//*.tf` matches all Terraform configuration files in the specified folder path.
 
+If the above specified format does not work please try the below format:
+
+- `git::ssh://git@github.com/test_org/test_repo//*.tf`
```
Comment (cbruno10) [inline docs/index.md:171]:
```suggestion
If the example formats above don't work for private repositories, this could be due to git credentials being stored by another tool, e.g., VS Code. An alternative format you can try is:

- `git::ssh://git@github.com/test_org/test_repo//*.tf`
```

File: docs/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbruno10) [reply docs/index.md:171]: Issues seemed to be due to VS Code storing git credentials, will suggest new doc text in a review

File: docs/index.md [hunk 3]
```diff
@@ -238,15 +242,8 @@ If the bucket is in another AWS account, the bucket policy will need to grant ac
...
-      "Action": [
-        "s3:ListBucket",
-        "s3:GetObject",
-        "s3:GetObjectVersion"
-      ],
-      "Resource": [
-        "arn:aws:s3:::test-bucket1",
-        "arn:aws:s3:::test-bucket1/*"
-      ]
+      "Action": ["s3:ListBucket", "s3:GetObject", "s3:GetObjectVersion"],
```
Comment (cbruno10) [inline docs/index.md:245]: Why was this changed? I'd prefer to leave it the same (especially if other index docs use the current format)

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@bigdatasourav Please see review comments, thanks!

File: docs/index.md (hunk 3 above)
Comment (bigdatasourav) [reply docs/index.md]: Due to my VS code setup, it got updated automatically after I saved the file. Reverted back.

File: docs/index.md (hunk 2 above)
Comment (bigdatasourav) [reply docs/index.md]: updated

**Review by cbruno10 — APPROVED** [review]
